tests/test_hypersphere.py

```diff
@@ -418,35 +417,32 @@ def test_inner_product(self):
 
         self.assertAllClose(expected, result)
 
-    @geomstats.tests.np_and_tf_only
     def test_variance(self):
         point = gs.array([0., 0., 0., 0., 1.])
-        points = gs.array([
-            point,
-            point])
+        points = gs.zeros((2, point.shape[0]))
+        points[0, :] = point
```

Comment:
These assignments make the tests fail in TF. Is it possible to rewrite without assignment in ``points"?

Reply:
Works now only in numpy and pytorch.